wgpu-native/src/swap_chain.rs

```diff
@@ -128,11 +128,16 @@ pub struct SwapChainOutput {
     pub view_id: TextureViewId,
 }
 
+#[derive(Debug)]
+pub enum SwapChainGetNextTextureError {
+    GpuProcessingTimeout = 1,
```

Comment:
not need to have `=1` there since this piece is not C facing

Reply:
I just did this so it's easier to convert to a value that is C facing [below](https://github.com/gfx-rs/wgpu/pull/369/files#diff-2a33782ea2561164d6e30cc67d01eefeR213)

Reply:
I see. This will not be needed if we keep returning `WGPUSwapChainOutput` instead of the error code.